### ISSUE TITLE
Aphantasia

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#11444]
**Preferred term label:**
Aphantasia

**Definition (free text, please give PubMed ID)**
Absence or near-absence of voluntary visual mental imagery, such that an individual is unable to intentionally generate visual images in the mind.
(PMID: 26115582, 39330760, 39386622)

**Parent term (use hpo.jax.org/app)**
Abnormality of mental imagery 
(a new term has been requested [here](https://github.com/obophenotype/human-phenotype-ontology/issues/11443))

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
@jagcsun added as HP:0020345
thanks!
